include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -40,20 +40,19 @@ DECLARE_DEFAULT_DELETER_FOR_ONNX_OBJECT(Allocator);
 DECLARE_DEFAULT_DELETER_FOR_ONNX_OBJECT(TensorTypeAndShapeInfo);
 DECLARE_DEFAULT_DELETER_FOR_ONNX_OBJECT(RunOptions);
 DECLARE_DEFAULT_DELETER_FOR_ONNX_OBJECT(SessionOptions);
-DECLARE_DEFAULT_DELETER_FOR_ONNX_OBJECT(ProviderFactoryPtr);
```

Comment:
Why do you delete this line?

Reply:
It originally caused an error because there is no ProviderFactory* but there is ProviderFactoryInterface*, I put it back.

Reply:
Did you put it back?